pkg/kubelet/apis/podresources/server_v1.go

```diff
@@ -66,11 +67,14 @@ func (p *v1PodResourcesServer) List(ctx context.Context, req *v1.ListPodResource
 
```

Comment:
If a Pod is in terminal phase, should returned list of the func contain the pod?

@Tal-or @ SergeyKanzhelev

Reply:
`List` return list of pods with their associate resources.
It's arguable whether terminal pods should be in that list, because eventually the managers (cpu-manager, memory-manager, etc.) reuses the resources of terminal pods.